aip/auth/4111.md

```diff
@@ -29,14 +29,36 @@ JWT as a means of authentication.
 Supporting self-signed JWT is considered a part of [application default
 credentials][2]. To understand the overall flow, please read [AIP-4110][2].
 
+### Scope vs. Audience
+
+In the past, the self-signed JWT is only applicable with `audience`. Recently,
+the usage of the "scope" claim in self-signed JWTs is now allowed in the Google
+Cloud auth ecosystem, which makes the self-signed JWT usable with both audience
+and scope. However, caller **must** not specify scope and audience at the same
+time to prevent potential conflicts.
+
+#### Opt-in Strategy
+
+Currently, this scope support for JWT is gated behind a configurable boolean
+option `UseJWTSelfSignWithScope`(the actual naming may vary depending on the
```

Comment:
In the auth libraries, we are using the term `JwtAccess` to represent these credentials. Do you think `UseJwtAccessWithScope` would be better, for consistency? I personally prefer it.

Reply:
SGTM